src/pages/consequences.module.scss

```diff
@@ -0,0 +1,56 @@
+@import '../assets/shared-styles/colors.module.scss';
+
+.riskBuying {
+    display: inline-block;
+    font-family: cera-light;
+    font-size:180%;
+    width:300px;
+}
+
+.offenderRegistry {
+    display: inline-block;
+    font-family: cera-bold;
+    font-size: 20px;
+    width: 300px;
+}
+
+.getArrested {
+    display: inline-block;
+    font-family: cera-bold;
+    font-size:20px;
+    padding-top: 50px;
+    padding-bottom: 20px;
+    width: 300px;
+}
+
+.payFines {
+    display: inline-block;
+    font-family: cera-bold;
+    font-size: 20px;
+    padding-top: 50px;
+    padding-bottom: 20px;
+    width: 309px;
+}
+
+.loseJob {
+    display: inline-block;
+    font-family: cera-bold;
+    font-size: 20px;
+    padding-top: 40px;
+    padding-bottom: 20px;
+    width: 312px;
+}
+
+.rethinkBuying {
+    display: inline-block;
+    font-size: 20px;
+    font-family: cera-bold;
+    padding-top: 20px;
+    width: 300px;
+}
+
+.readLaws {
+    height: 27px;
```

Comment:
Again, height & width should not be set on text based elements--use the padding and margin properties to manage how much room they take up on the screen

Reply:
Removed height